builder/FindPackages.cmake

```diff
@@ -494,7 +494,6 @@ if( Linux )
   pkg_check_modules( PKG_MFX       libmfx>=1.28 )
 
   if ( PKG_X11_FOUND AND PKG_LIBVA_X11_FOUND )
```

Comment:
If user explicitly sets ENABLE_X11=OFF then why we check X11 related packages? Please, modify the code as:
```
if(ENABLE_X11)
  pkg_check_modules( PKG_X11       x11 )
  pkg_check_modules( PKG_LIBVA_X11 libva-x11>=0.33 )
endif()

```